utkdigitalinitiatives/canopy
# Fix problems created by DIGITAL-1222.

## What Does This Do

Corrects several problems:

1. If rights doesn't exist, it sets to use "Copyright Not Evaluated" instead of throwing error.
2. Let's you use any known rights statement.

## How Can I Test

1. Go to the netlify link below:
    - Test that you see rights statements in the viewer for "Whaley," "Ludden," and "Jimenez". You can use search to limit. All three should show different badges.


## What to do if I approve

If you approve without suggestions, click approve then merge.  Don't wait for multiple reviewers for merge.
